src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -109,13 +148,24 @@ public override void PressesBegan(NSSet<UIPress> presses, UIPressesEvent evt)
 					}
 				}
 			}
-
+#endif
 			if (!handled)
 			{
 				base.PressesBegan(presses, evt);
 			}
 		}
 
+		public void SetOwner(CoreWindow owner)
+		{
+			_owner = new WeakReference<CoreWindow>(owner);
+			_ownerEvents = (ICoreWindowEvents)owner;
+		}
+
+		public CoreWindow GetOwner() =>
+			_owner != null && _owner.TryGetTarget(out var target) && target is { }
+			? target
+			: null;
```

Comment:
Does this need to be public?

Reply:
The `GetOwner()` method? not necessarily. I made it public to match other classes already using this pattern in the repository (to keep consistency). But it can definitely be private.

Reply:
Let's make it internal for now then.

Reply:
Removed method not longer needed